packages/edit-site/src/components/template-part-converter/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { BlockSettingsMenuControls } from '@wordpress/block-editor';
+import { MenuItem } from '@wordpress/components';
+import { switchToBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+export default function TemplatePartConverter() {
```

Comment:
Should this live in the `block-editor` package? It seems like it would fit well there, since it's basically part of the block menu. (maybe we could then provide a mechanism to turn it off for the post editor?)

Reply:
I would assume that if we want this in both editors (site and post) when FSE is enabled, that this would be better in `block-editor` as well.  However, if it doesn't belong in the post editor I would think this is a fitting location for it?  (asked about this below)

Reply:
It looks like the idea is to keep it edit-site only (for now at least).  Id say edit-site package makes sense, and we can move it over to block-editor if things change and we want this in the post editor as well.